src/lib/setup.js

```diff
@@ -220,7 +222,7 @@ export default function(app) {
     })
 
     mountAndReload(path.resolve('src/desktop'), {
-      watchModules: ['@artsy/reaction', '@artsy/stitch'],
+      watchModules: ['@artsy/reaction'],
```

Comment:
/cc @damassi ^

Reply:
Thnx @erikdstock - this `watchModules` field should be used sparingly lest someone accidentally wire up a local version of a module that is in an old state. I realized I had left stitch in there from before when we were SHing earlier.